packages/blockchain/test/index.spec.ts

```diff
@@ -596,8 +599,9 @@ tape('blockchain test', (t) => {
   })
 
   t.test('should put multiple blocks at once', async (st) => {
+    const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Chainstart })
     const blocks: Block[] = []
-    const genesisBlock = Block.genesis({ header: { gasLimit: 8000000 } })
```

Comment:
heh yeah since the block number still defaults to zero, we weren't losing much, as there were not very many cases were we needed the actual genesis params.